### PR TITLE
Add release pipeline for gpdb7

### PR DESCRIPTION
**Pull request by zhaorui20005**
The release pipeline is :https://dev2.ci.gpdb.pivotal.io/teams/gp-dataflow/pipelines/pgbouncer_gpdb7_release

### CODE REVIEW

File: concourse/pipelines/README.md [hunk 1]
```diff
@@ -16,4 +18,6 @@ The pipelines are run by the following command.
 
 `fly -t dev2 set-pipeline -p pgbouncer_gpdb5_pr -c ./bouncer_gpdb5_pr.yml`
```
Comment (hyongtao-db) [inline concourse/pipelines/README.md:20]: Add
`fly -t dev2 set-pipeline -p bouncer_gpdb6_and_gpdb7_pr -c ./bouncer_gpdb6_and_gpdb7_pr`

**Review by hyongtao-db — APPROVED** [review]
LGTM

File: concourse/pipelines/README.md (hunk 1 above)
Comment (zhaorui20005) [reply concourse/pipelines/README.md:20]: It's at line 15 which has not be shown by default.

File: concourse/scripts/install_gpdb_component [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+if [[ $1 == "--help" ]]; then
+    echo "Make sure GPHOME environment variable is set, then run this file to install pgbouncer"
+    exit 0
+fi
+if [[ -z "${GPHOME}" ]]; then
+    echo "Error: Environment Variable GPHOME not set!!"
+    exit 1
+fi
+
+cp -av bin share $GPHOME
```
Comment (huiliang-liu) [inline concourse/scripts/install_gpdb_component:14]: do we have `share` folder too?

Comment (zhaorui20005) [reply concourse/scripts/install_gpdb_component:14]: Yes, share dir contains docs of pgbouncer.

**Review by huiliang-liu — APPROVED** [review]

**Pull request merged**